guides/release/components/built-in-components.md

```diff
@@ -98,13 +99,23 @@ Checkboxes support the following properties:
 Which can be bound or set as described in the previous section.
 
 
-Checkboxes are a special input type. If you want to dispatch an action on a certain [event](https://api.emberjs.com/ember/release/classes/Component#event-handler-methods), you will always need to define the event name in camelCase format:
+Checkboxes are a special input type.  If you want to dispatch an action on a certain [event](https://api.emberjs.com/ember/release/classes/Component#event-handler-methods),
+you will always need to either define the event name in camelCase format (e.g. `@keyDown`), or
+use an `on` helper with the [Web-API event name](https://developer.mozilla.org/en-US/docs/Web/API/Document/keydown_event) (e.g. `on 'keydown'`):
 
 ```handlebars
 <label for="firstname">First Name</label>
-<Input @type="checkbox" @key-press={{this.updateName}} id="firstname" />
+{{!-- This does not work: uses dasherized event name --}}
+<Input @type="checkbox" @key-down={{this.updateName}} id="firstname" />
+{{!-- This does not work: uses actual event name --}}
+<Input @type="checkbox" @keydown={{this.updateName}} id="firstname" />
+{{!-- This works: uses camelCase event name--}}
+<Input @type="checkbox" @keyDown={{this.updateName}} id="firstname" />
+{{!-- This works: uses 'on' with actual event name--}}
+<Input @type="checkbox" {{on 'keydown' this.updateName}} id="firstname" />
```

Comment:
```suggestion
<Input @type="checkbox" {{on "keydown" this.updateName}} id="firstname" />
```
The default linter enforces using double quotes.

Can we move the working examples up to the top of this list?

Reply:
will do

Reply:
done. Lemme know if you'd like me to rebase and squash at any point.